specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{Model Id}",
+    "body": {}
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{endpoint}/formrecognizer/v2.0-preview/custom/models/{modelId}/analyzeResults/{resultId}"
```

Comment:
I assume this is a copy of examples from 2.1-preview.1.  If so, same comments as above.  Need to change URLs to v2.1-preview.2 and the response version to 2.1.2.

Reply:
done.